src/cli/argument-parser.js

```diff
@@ -107,6 +107,7 @@ export default class CLIArgumentParser {
             .option('--ports <port1,port2>', 'specify custom port numbers')
             .option('--hostname <name>', 'specify the hostname')
             .option('--proxy <host>', 'specify the host of the proxy server')
+            .option('--proxy-bypass <url1,url2>', 'specify the urls to bypass the proxy server')
```

Comment:
```
            .option('--proxy-bypass <rules>', 'specify a comma-separated list of rules that define URLs accessed bypassing the proxy server')
```

Reply:
Regarding https://github.com/DevExpress/testcafe/pull/2187#discussion_r172121922 we can just add `Rules consist of URLs with wildcards enabled`.
Is that simpler. @AndreyBelym ?